examples/plugins/echo/main.go

```diff
@@ -0,0 +1,63 @@
+package echo
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/aerogear/charmil/pkg/factory"
+	"github.com/aerogear/charmil/pkg/localize"
+	"github.com/spf13/cobra"
+	"golang.org/x/text/language"
+)
+
+func EchoCommand() (*cobra.Command, error) {
+	var (
+		iter                 int
+		uppercase, lowercase bool
+	)
+
+	locConfig := localize.Config{
+		Language: language.English,
+		Path:     "examples/plugins/echo/locals/en/echo.en.yaml",
+		Format:   "yaml",
+	}
+
+	loc, err := localize.InitLocalizer(locConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	f := factory.Default(loc)
+
+	echoCmd := &cobra.Command{
+		Use:          f.Localizer.LocalizeByID("echo.cmd.use"),
+		Short:        f.Localizer.LocalizeByID("echo.cmd.short"),
+		Example:      f.Localizer.LocalizeByID("echo.cmd.example"),
+		SilenceUsage: true,
+		Args:         cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			message := strings.Join(args, " ")
+
+			if uppercase {
```

Comment:
why this logic?

Reply:
That's just a part of an example plugin that I've added and doesn't do much.
I wanted to have a basic and minimal single command example plugin, so I chose to add this. 😅

All it does is clone the functionality of the Linux `echo` command ie. print all the strings passed as arguments, into stdout.
It has 3 local flags:
- **uppercase**: If this is enabled, strings are converted to uppercase before printing into stdout
- **lowercase**: If this is enabled, strings are converted to lowercase before printing into stdout
- **repeat**: The value that is passed into this flag is the number of times the input strings will be printed to stdout

Reply:
As we discussed I think it is best for examples to be minimal.
Less non functional code then less things that people need to read